apps/src/templates/teacherHomepage/CollapsibleSection.jsx

```diff
@@ -58,7 +58,8 @@ const CollapsibleSection = React.createClass({
     header: React.PropTypes.string.isRequired,
     linkText: React.PropTypes.string,
     link: React.PropTypes.string,
-    collapsible: React.PropTypes.bool
+    collapsible: React.PropTypes.bool,
+    showLink: React.PropTypes.bool
```

Comment:
Sort of - the way CollapsibleSection is being used in RecentCourses means the same link is always passed to the CollapsibleSection from RecentCourses, but when one uses the RecentCourses component they need the option to show that link or not.